packages/holocron/docs/api/README.md

```diff
@@ -103,29 +103,47 @@ The optional `holocron` object set to the parent React Component inside a Holocr
 ```jsx
 import React from 'react';
 import PropTypes from 'prop-types';
-import { reducer, fetchData } from '../duck';
+import { fromJS } from 'immutable';
+
+const HelloWorld = ({ moduleState: { name }, moduleLoadStatus }) => {
+  if (moduleLoadStatus === 'loading') return <h1>Loading...</h1>;
+  if (moduleLoadStatus === 'error') return <h1>Error!</h1>;
+  return <h1>Hello, {name}!</h1>;
+};
 
-const HelloWorld = ({ moduleState: { myData } }) => (
-  <h1>
-Hello,
-    {myData.name}
-  </h1>
-);
 HelloWorld.propTypes = {
   moduleState: PropTypes.shape({
-    myData: PropTypes.string,
+    name: PropTypes.string,
   }).isRequired,
+  moduleLoadStatus: PropTypes.oneOf(['loading', 'loaded', 'error']),
 };
 
-const loadModuleData = ({ store: { dispatch }, ownProps }) => dispatch(fetchData(ownProps.input));
-const shouldModuleReload = (oldProps, newProps) => oldProps.input !== newProps.input;
+const loadModuleData = ({ store: { dispatch } }) => dispatch({ type: 'SET_NAME', name: 'World' });
+
+const shouldModuleReload = (oldProps, newProps) => oldProps.moduleState?.name !== newProps.moduleState?.name;
+
+// This reducer is supplying 'moduleState' in our component
+// Note: reducers use immutable.js
+const reducer = (state = fromJS({}), action) => {
+  switch (action.type) {
+    case 'SET_NAME': {
+      return state.set('name', action.name);
+    }
+    default: return state;
+  }
+};
 
 HelloWorld.holocron = {
-  name: 'hello-world',
+  name: 'holocron-hello-world',
   reducer,
   loadModuleData,
   shouldModuleReload,
-  options,
+  // options,
+};
+
+if (!global.BROWSER) {
+  // eslint-disable-next-line global-require
+  HelloWorld.appConfig = require('../appConfig').default;
 }
```

Comment:
`appConfig` is specific to One App. One App can't work without holocron but holocron is technically not reliant on One App. There is a lot of coupling between the two in real terms but we want to try and keep Holocron decoupled from one app.  

```suggestion
```

Reply:
My fault! I will remove that.